2021-11-09 22:33:48,946 INFO    MainThread:32181 [wandb_setup.py:_flush():71] setting env: {}
2021-11-09 22:33:48,946 INFO    MainThread:32181 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-09 22:33:48,946 INFO    MainThread:32181 [wandb_init.py:_log_setup():357] Logging user logs to /home/younesbelkada/Travail/MVA/Recvis/Assignment3/recvis21_a3/wandb/run-20211109_223348-1hr13y9m/logs/debug.log
2021-11-09 22:33:48,946 INFO    MainThread:32181 [wandb_init.py:_log_setup():358] Logging internal logs to /home/younesbelkada/Travail/MVA/Recvis/Assignment3/recvis21_a3/wandb/run-20211109_223348-1hr13y9m/logs/debug-internal.log
2021-11-09 22:33:48,947 INFO    MainThread:32181 [wandb_init.py:init():390] calling init triggers
2021-11-09 22:33:48,947 INFO    MainThread:32181 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-11-09 22:33:48,947 INFO    MainThread:32181 [wandb_init.py:init():435] starting backend
2021-11-09 22:33:48,947 INFO    MainThread:32181 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-09 22:33:48,965 INFO    MainThread:32181 [backend.py:ensure_launched():199] starting backend process...
2021-11-09 22:33:49,007 INFO    MainThread:32181 [backend.py:ensure_launched():205] started backend process with pid: 32204
2021-11-09 22:33:49,017 INFO    MainThread:32181 [wandb_init.py:init():444] backend started and connected
2021-11-09 22:33:49,020 INFO    MainThread:32181 [wandb_init.py:init():503] updated telemetry
2021-11-09 22:33:49,037 INFO    MainThread:32181 [wandb_init.py:init():533] communicating current version
2021-11-09 22:33:49,765 INFO    MainThread:32181 [wandb_init.py:init():538] got version response 
2021-11-09 22:33:49,765 INFO    MainThread:32181 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-09 22:33:50,082 INFO    MainThread:32181 [wandb_init.py:init():576] starting run threads in backend
2021-11-09 22:33:50,123 INFO    MainThread:32181 [wandb_run.py:_console_start():1693] atexit reg
2021-11-09 22:33:50,124 INFO    MainThread:32181 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-11-09 22:33:50,124 INFO    MainThread:32181 [wandb_run.py:_redirect():1572] Redirecting console.
2021-11-09 22:33:50,125 INFO    MainThread:32181 [wandb_run.py:_redirect():1628] Redirects installed.
2021-11-09 22:33:50,125 INFO    MainThread:32181 [wandb_init.py:init():603] run started, returning control to user process
2021-11-09 22:33:50,127 INFO    MainThread:32181 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-11-09 22:33:50,127 INFO    MainThread:32181 [wandb_run.py:_restore():1635] restore
2021-11-09 22:33:52,635 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 814
}

2021-11-09 22:33:52,995 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 814
}

2021-11-09 22:33:53,116 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 1818
}

2021-11-09 22:33:53,219 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 1844
}

2021-11-09 22:33:53,322 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 1844
}

2021-11-09 22:33:53,426 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 814
  total_bytes: 1844
}

2021-11-09 22:33:53,529 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 955
  total_bytes: 1844
}

2021-11-09 22:33:53,632 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1844
  total_bytes: 1844
}

2021-11-09 22:33:53,735 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1844
  total_bytes: 1844
}

2021-11-09 22:33:53,837 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1844
  total_bytes: 1844
}

2021-11-09 22:33:53,940 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1844
  total_bytes: 1844
}

2021-11-09 22:33:54,043 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1844
  total_bytes: 1844
}

2021-11-09 22:33:54,508 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1844
  total_bytes: 1844
}

2021-11-09 22:33:55,187 INFO    MainThread:32181 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1844
  total_bytes: 1844
}
local_info {
}

2021-11-09 22:33:56,289 INFO    MainThread:32181 [wandb_run.py:_append_files():2061] logging synced files
